tests/test_metrics.py

```diff
@@ -0,0 +1,102 @@
+import numpy as np
+import pytest
+
+from continuum import MetricsLogger
+
+# yapf: disable
+
+@pytest.fixture
+def numpy_data():
+    nb_classes = 20
+    nb_tasks = 5
+    nb_data = 100
+    seen_classes = 0
+    inc_classes = nb_classes // nb_tasks
+
+    targets = []
+    tasks = []
+
+    for t in range(nb_tasks):
+        seen_classes += inc_classes
+
+        task_targets = np.concatenate([
+            np.ones(nb_data) * c
+            for c in range(seen_classes)
+        ])
+        task_tasks = np.concatenate([
+            np.ones(nb_data * inc_classes) * tt
+            for tt in range(t + 1)
+        ])  # We also see previous data
+
+        targets.append(task_targets)
+        tasks.append(task_tasks)
+
+    return targets, tasks
+
+
+@pytest.mark.parametrize("mode,expected", [
+    ("best", 1.), ("worst", 0.), ("random", None)
+])
+def test_metrics(numpy_data, mode, expected):
+    logger = MetricsLogger()
+
+    all_targets, all_tasks = numpy_data
+
+    for targets, task_ids in zip(all_targets, all_tasks):
+        if mode == "best":
+            preds = np.copy(targets)
+        elif mode == "worst":
+            # Trick to never generate the correct predictions
+            # only work for more three classes or more
```

Comment:
something is wring with this comment I think

Reply:
Indeed... it should work, i'll check it. Thx!